src/components/Common/common.css

```diff
@@ -1,3 +1,21 @@
-p {
-    @apply lil-font-medium lil-text-font-200;
+.common-padding {
+  @apply lil-py-8;
+}
+
+@screen lg {
+  .full-section {
+    @apply lil-h-screen;
+  }
+  .common-padding {
+    @apply lil-py-24;
+  }
+}
+
+@screen xl {
+  .full-section {
+    @apply lil-h-screen;
+  }
+  .common-padding {
+    @apply lil-py-24;
+  }
```

Comment:
i think we shud not expose them as separate classes as this may not be only for our static pages styling in future